src/index.js

```diff
@@ -0,0 +1,16 @@
+const tetrispiece = [
+    { shape: [[1,1,1,1]], colour: 'cerulean blue'},
+    { shape: [[1,1], [1,1]], colour: 'honey yellow'},
+    { shape: [[1,1,1], [1,0,0]], colour: 'flurescent pink'},
+    { shape: [[1,1,1], [0,0,1]], colour: 'brick red'},
+    { shape: [[1,1,1], [0,1,0]], colour: 'mint green'},   
+    { shape: [[1,1,0], [0,1,1]], colour: 'bauge'},
+    { shape: [[0,1,1], [1,1,0]], colour: 'rose gold'}
+];
+function getrandompiece(){
+    const randomint = Math.floor(Math.random()*tetrispiece.length);
```

Comment:
Perhaps this can have a more meaningful name; perhaps to include that this indexes the list?
```suggestion
    const randomIndex = Math.floor(Math.random()*tetrispiece.length);
```